File: pset6/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height > 0 and height <= 8:
        break

for i in range(height):
    for j in range(height):
        if j < height - 1 - i:
            print(" ", end="")
        else:
            print("#", end="")

    print("  ", end="")

    for k in range(i + 1):
        print("#", end="")

    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Receiving de height and testing the input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //designing the pyramid
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (j < n - 1 - i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}